### ISSUE TITLE
Genotype Cohort

### ISSUE THREAD

**Issue by andremrsantos** [google/deepvariant#83]
Hi guys, 

I have been doing some test on DeepVariant genotyping call. I run the suit and got very good results, but as part of experiments I need to generate a cohort VCF (multi-sample). As suggested here on github, I generated GVCF for all my samples, but when I tried to use GATK's  CombineGVCFs or GenotypeGVCFs neither worked because they don't recognize the alternative allele `<*>`, instead GATK moved to `<NON_REF>` on the more recent versions.
After identified the problem, I run a simple substitution using `sed` to replace all occurrences of `<*>` for `<NON_REF>` and the commands ran fine. 

`zcat SAMPLE.deepvar.g.vcf.gz | sed 's/<*>/<NON_REF>/g' | bgzip -c > SAMPLE.gvcf.gz`

May I suggest this update for your software to keep the compatibility with GATK?

Best,
André Santos

**Comment by pichuan** [comment]
Hi,
I understand that it's a common use case to be compatible with GATK. We'll consider potentially adding a flag for that conversion. But since we're following the spec (using the VCF v4.3 spec: page 25 of this doc https://samtools.github.io/hts-specs/VCFv4.3.pdf), this won't be of high priority. For now the simple substitution you're doing is correct.

I filed a bug internally to track. I'll close this issue now.

**Comment by tedyun** [comment]
Please note that "*" character must be escaped in the above script by @andremrsantos . Please use the following script for converting g.vcf.gz file (and for indexing).

`zcat dv.g.vcf.gz | sed -e 's/<\*>/<NON_REF>/g' | bgzip -c > dv.gatkcompliant.g.vcf.gz && tabix -p vcf dv.gatkcompliant.g.vcf.gz`

This will generate `dv.gatkcompliant.g.vcf.gz` from `dv.g.vcf.gz`

**Comment by AndrewCarroll** [comment]
Hi @andremrsantos

I wanted to update this issue with recent developments for cohort merging. With the DeepVariant v0.9 release, we recommend Best practices for multi-sample variant calling with DeepVariant

We encourage you or other users interested in multi-sample calling to follow these recommendations to combine multiple DeepVariant gVCFs. In our benchmarks, we found this merging approach more accurate than merging with GATK GenotypeGVCFs (and we found merging DeepVariant gVCFs using GATK GenotypeGVCFs substantially less accurate than the single-sample DeepVariant VCFs themselves).

Thank you.
